mozci/console/commands/push.py

```diff
@@ -587,6 +588,20 @@ def handle(self) -> None:
                     f"<comment>Push {push.branch}/{push.rev} was backedout and all of its failures and the ones of its children were marked as intermittent or marked as caused by another push</comment>"
                 )
 
+            for task in p.tasks:
```

Comment:
```suggestion
            for task in push.tasks:
```

Otherwise this line will raise an error:
```
  UnboundLocalError

  local variable 'p' referenced before assignment
```

Reply:
Oops, I copied and pasted from a separate script that I wrote without then actually testing the command :P
Relying too much on linting to catch these errors!